client/html/public/ticket.html

```diff
@@ -24,6 +23,12 @@
                 document.getElementById("ticket-resolve-button").textContent =
                     `${ticket.resolved ? "Unresolve" : "Resolve"} ticket`;
 
+                if (response.status !== 200) {
+                    const messageContainer = document.getElementById("message");
+                    messageContainer.innerHTML = `\
+        <span class="update-status">Error: user unauthorized access ticket: ${ticket.reason || 'no reason specified'}</span>`;
```

Comment:
🤔 I feel like there's missing something in this...
```suggestion
        <span class="update-status">Error: user unauthorized to access ticket: ${ticket.reason || 'no reason specified'}</span>`;
```
Maybe?